lib/gen_stage.ex

```diff
@@ -1106,6 +1106,11 @@ defmodule GenStage do
       cancellations, the reason is wrapped in a `:cancel` tuple.
     * `:min_demand` - the minimum demand for this subscription
     * `:max_demand` - the maximum demand for this subscription
+    * `:selector` - a function (event :: any -> boolean)
+                    which is used in the dispatcher to filter events
```

Comment:
And we shouldn't document this here but in the broadcast dispatcher module docs. :)